clusters/prow-build/monitoring-build-kustomization.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: kustomize.toolkit.fluxcd.io/v1
+kind: Kustomization
+metadata:
+  name: monitoring-build
```

Comment:
In our setup, there is a single `monitoring` `Kustomization` in `clusters/base` and a patch in both `clusters/prow-{build,trusted}` that overwrite `spec.path` to `deploy/monitoring/{build,trusted}`.
This way, the main part of the `Kustomization` is shared/reused and both have the same name for consistency.

This doesn't mean we have to do this here in the same way. Just some ideas :)

Reply:
Done 😄